ui/app/components/splash-page.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{yield to="altContent"}}
```

Comment:
Previous behavior yielded this if it exists, and otherwise shows the rest of the template. Seems like this will render the rest of the template either way?